vm/systemSmartContracts/validator.go

```diff
@@ -175,12 +191,22 @@ func NewValidatorSmartContract(
 		enableUnbondTokensV2Epoch:        args.EpochConfig.EnableEpochs.UnbondTokensV2EnableEpoch,
 		validatorToDelegationEnableEpoch: args.EpochConfig.EnableEpochs.ValidatorToDelegationEnableEpoch,
 		shardCoordinator:                 args.ShardCoordinator,
+		stakeLimitsEnableEpoch:           args.EpochConfig.EnableEpochs.StakeLimitsEnableEpoch,
+		nodeLimitPercentage:              args.StakingSCConfig.NodeLimitPercentage / 100.0,
+		nodesCoordinator:                 args.NodesCoordinator,
+	}
+
+	reg.totalStakeLimit = core.GetIntTrimmedPercentageOfValue(args.GenesisTotalSupply, args.StakingSCConfig.StakeLimitPercentage)
```

Comment:
One idea spread in the office is to drop the stake limitation on the SP and only keep the node percentage limitation. In this way the "market" for the SPs will equalize itself while keeping the ability to absorb unstaked funds.

Reply:
I pretty much like "not getting involved" in office ideas.

Reply:
After further discussions, can be left as it is in order to have some implemented control over the stake. The percentage can be later established.